docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx

```diff
@@ -3,15 +3,15 @@ id: sign-in-with-github-google-facebook-linkedin
 title: Sign in with GitHub, Google, Facebook, LinkedIn, ...
 ---
 
-In this document we will take a look at setting up "Sign in with GitHub" using
+In this document we will take a look at setting up "Sign in with GitHub/Google" using
 ORY Kratos.
 
 Run the [Quickstart](../quickstart.mdx) with Docker Compose:
 
 ```shell script
 # This assumes that you have ORY Kratos checked out locally and
 # that your current directory ("pwd") is the folder where ORY Kratos
-# is.
+# is. Note that it is only tested against `v0.3.0-alpha.1` tag.
```

Comment:
This should work in future versions as well so we don't need to add this here :)

```suggestion
# is.
```